ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/dialogs/PluginSelectionDialog.java

```diff
@@ -153,8 +159,8 @@ private static IPluginModelBase[] getElements(boolean includeFragments) {
 		return PluginRegistry.getActiveModels(includeFragments);
 	}
 
-	public static HashSet<String> getExistingImports(IPluginModelBase model, boolean includeImportPkg) {
-		HashSet<String> existingImports = new HashSet<>();
+	public static HashMap<String, Boolean> getExistingImports(IPluginModelBase model, boolean includeImportPkg) {
+		HashMap<String, Boolean> existingImports = new HashMap<>();
```

Comment:
Maybe I oversaw something, but I can only see the key-set of the map being used and values seem not to be considered at all. If that's correct, can this `Map` stay a `Set`?